docs/SRS/SRS.tex

```diff
@@ -453,52 +453,46 @@ \subsection{Individual Product Use Cases (PUC's)}
-  \item \textbf{Requirement:} The tool must report to the user any discrepancies between the original and suggested refactored code.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} Discrepancy reports to the user clearly highlight differences in outputs
-  \end{itemize}
-  \item \textbf{Requirement:} The tool must allow users to input their original test suite as a required argument.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} Users can specify a path to their test suite, and the tool recognizes and utilizes it for testing the refactored code.
-  \end{itemize}
-  \item \textbf{Requirement:} The tool must ensure that the original functionality of the code is preserved after refactoring.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} The tool runs the original test suite against the refactored code and passes 100\% of the tests.
-  \end{itemize}
-  \item \textbf{Requirement:} The tool must be compatible with various Python versions and common libraries.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} The tool operates correctly with the latest two major versions of Python (e.g., Python 3.8 and 3.9) and commonly used libraries.
-  \end{itemize}
-  \item \textbf{Requirement:} The tool must generate comprehensive reports on detected smells, refactorings applied, energy consumption measurements, and testing results.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:}  Reports are clear, well-structured, and provide actionable insights, with users able to easily understand the results.
-  \end{itemize}
-  \item \textbf{Requirement:} The tool must provide comprehensive documentation and help resources.
-  \begin{itemize}[label={}]
-      \item \textbf{Fit Criteria:} Documentation covers installation, usage, and troubleshooting, receiving positive feedback for clarity and completeness from users.
-  \end{itemize}
+\begin{enumerate}[label=FR \arabic*., wide=0pt, leftmargin=*]
+    \item \emph{The system must accept Python source code files.}\\
+    {\bf Rationale:} The system needs to process Python code as its primary input to refactor and improve energy efficiency.\\
+    {\bf Fit Criterion:} The system successfully processes valid Python files without errors and provides feedback for invalid files.
+    \item \emph{The system must identify specific code smells that can be targeted for energy saving.}\\
+    {\bf Rationale:} Energy inefficiencies are often related to well-known code smells, so identifying them is the first step in improving efficiency.\\
+    {\bf Fit Criterion:} The tool should detect and report at least 80\% of the following code smells: Large Class (LC), Long Parameter List (LPL), Long Method (LM), Long Message Chain (LMC), Long Scope Chaining (LSC), Long Base Class List (LBCL), Useless Exception Handling (UEH), Long Lambda Function (LLF), Complex List Comprehension (CLC), Long Element Chain (LEC), Long Ternary Conditional Expression (LTCE).
```

Comment:
References added?

Reply:
Yeah, it looks like this needs references, I will add them in a different PR